.github/workflows/checks.yml

```diff
@@ -162,3 +162,25 @@ jobs:
           name: coverage-python${{ matrix.python-version }}-fast
           path: .coverage
           include-hidden-files: true
+
+
+  Validate-SLC-Github-Workflows:
```

Comment:
That might be useful for the SLC repos, too. Maybe we should extract it into a workflow call or a workflow which we also generate.